src/utils/buildPieData.js

```diff
@@ -12,7 +12,7 @@ export function buildPieData(rawData) {
     const duration = session.duration / 60000; // ms to minutes
     acc[`${duration} minutes`].sessions += 1;
     return acc;
-  }, durationShell);
+  }, initialDurationData);
```

Comment:
I like this naming change. For consistency, let's also rename other instances of similar variable names (for example, renaming `attendanceShell` to `initialAttendanceData`) across the codebase.

Reply:
On it